composer.json

```diff
@@ -17,18 +17,19 @@
     ],
     "require": {
         "php": ">=7.1",
-        "symfony/yaml": "~2.3||~3.0||~4.0||^5.0",
-        "symfony/console": "~2.4||~3.0||^4.0",
-        "symfony/finder": "~2.3||~3.0||~4.0||^5.0",
-        "symfony/validator": "~2.3||~3.0.0||~3.1.0||^3.2.4||~4.0.0||~4.1.0",
-        "symfony/filesystem": "~2.3||~3.0||~4.0||^5.0",
-        "symfony/config": "~2.3||~3.0||^4.0",
+        "symfony/yaml": "^2.7.0||^3.0.0||^4.0.0||^5.0.0",
+        "symfony/config": "^2.7.0||^3.0.0||^4.0.0||^5.0.0",
+        "symfony/console": "^2.7.0||^3.0.0||^4.0.0||^5.0.0",
+        "symfony/filesystem": "^2.7.0||^3.0.0||^4.0.0||^5.0.0",
+        "symfony/finder": "^2.7.0||^3.0.0||^4.0.0||^5.0.0",
+        "symfony/translation": "^2.7.0||^3.0.0||^4.0.0||^5.0.0",
+        "symfony/validator": "^2.7.0||~3.1.7||^3.2.4||^4.0.0",
```

Comment:
^4 works now, but it appears there's a bunch of issues introduced in ^5.

Reply:
Why not moving to https://github.com/propelorm/Propel2/pull/1590 as we discussed?
That would make it much easier for you to keep the leftover range for those dependencies working no?

Reply:
2.7 was the easiest to fix.

Reply:
Well, if it isn't too much trouble keeping 2.x support, then sure, all good.